mne_bids/tsv_handler.py

```diff
@@ -44,6 +44,51 @@ def _combine(data1, data2, drop_column=None):
     return data
 
 
+def _combine_cols(data1, data2, key_order, col_name):
+    """Add two OrderedDict's together and optionally drop repeated data.
+
+    Parameters
+    ----------
+    data1 : collections.OrderedDict
+        Original OrderedDict.
+    data2 : collections.OrderedDict
+        New OrderedDict to be added to the original.
+    key_order : list
+        List of original dictionary keys to maintain the
+        original ordering.
+    col_name : str
+        Name of the column to index over (e.g. `participant_id`
+        in participants.tsv file).
+
+    Returns
+    -------
+    data : collections.OrderedDict
+        The new combined data.
+
+    """
+    data = deepcopy(data2)
+
+    # keep the original order, and add back
+    # the data in participants files that user added
+    for key in data1.keys():
+        # skip this key if mne-bids handles it
+        if key in key_order:
+            continue
+
+        # add them back per participant id
```

Comment:
participant id? your comments don't make sense

can you clean up a bit and then ping me again? adding some simple examples in the docstring might help. See the tests for _combine_rows and [docstrings in MNE-Python with examples](https://github.com/mne-tools/mne-python/blob/master/mne/event.py#L623-L649)

Reply:
Sorry I was bit hasty and this was leftover code from when it was just inside `_participants_tsv()` function.  

I added some examples to show what this is intended to do.